src/admin/components/forms/field-types/Select/Input.tsx

```diff
@@ -62,9 +62,14 @@ const SelectInput: React.FC<SelectInputProps> = (props) => {
   let valueToRender;
 
   if (hasMany && Array.isArray(value)) {
-    valueToRender = value.map((val) => options.find((option) => option.value === val));
+    valueToRender = value.map((val) => {
+      const found = (options.find((option) => option.value === val));
+      found.label = getTranslation(found.label, i18n);
```

Comment:
Do we want to ensure found exists before accessing found.label? Technically it could be undefined if not found.